docs/pages/access-controls/access-requests/role-requests.mdx

```diff
@@ -56,138 +48,146 @@ metadata:
   name: dba
 spec:
   allow:
-    # ...
-  deny:
-    # ...
+    db_labels:
+      '*': '*'
   options:
-    # ...
     # Only allows the contractor to use this role for 1 hour from time of request.
     max_session_ttl: 1h
 ```
 
-**Admin Role**
-This role allows the admin to approve the contractor's request.
+**Approver Role**
+
+This role allows users to approve requests for the `dba` role.
 
 ```yaml
 kind: role
 version: v5
 metadata:
-  name: admin
+  name: approver
 spec:
   allow:
     # `review_requests` permits the listed roles to be approved
     review_requests:
       roles:
       - 'dba'
-    rules:
-    # `access_request` is part of Access Workflows introduced in 4.2
-    # `access_request` should only be given to Teleport Admins.
-    # For a full list of allow-rules, see
-    # https://goteleport.com/docs/access-controls/reference/
-    - resources:
-      - access_request
-      verbs:
-      - list
-      - read
-      - update
-      - delete
-  deny:
-    # ...
-  options:
-    # ...
 ```
 
-```code
-$ tsh login --request-roles=dba teleport-cluster
-# Seeking request approval... (id: bc8ca931-fec9-4b15-9a6f-20c13c5641a9)
-```
+<Admonition type="warning" title="Requires Teleport Enterprise">
+Roles containing a `review_requests` rule can only be used in Teleport
+Enterprise. In OSS Teleport, Access Requests must be approved by an admin
+running `tctl`.
+</Admonition>
+
+## Requesting Access
+
+While Teleport Enterprise supports the same CLI-based workflows for requesting
+access to roles, most users will prefer to request access via the web UI.
```

Comment:
Note to reviewers: I didn't cover the `tsh` flow for requesting access from the command line. Wasn't sure if I should duplicate a bunch of that info here or link to that page in some way, so I just briefly mentioned it here.

Reply:
If the `tsh` flow is what you're referencing with "the same CLI-based workflow", then we need to mention it somehow. That "the same" isn't referencing anything previously discussed.

Reply:
It's previously discussed in the admonition 5-10 lines above?